### DISCUSSION TITLE
Outreach: Energy conferences

### DISCUSSION THREAD

**Discussion by clizbe** [TulipaEnergy/TulipaEnergyModel.jl#78] [Ideas]
Saw this article comparing different energy modelling tools:
https://doi.org/10.1016/j.esr.2023.101136

It's worth a read to see what the general user community is looking for.

It also mentions OpenMod, which appears to be a wiki about energy models.
Once we have something decent to show, we could leverage it to boost our user community and get interaction.
https://openmod-initiative.org/

**Comment by suvayu** [comment]
OpenMod also has a reasonably active [forum](https://forum.openmod.org/).

**Comment by datejada** [comment]
They have a forum that might be interesting to attend for next year(s):

https://www.linkedin.com/posts/maximilian-parzen-b047a1126_openmod-stanford-energymodelling-activity-7109522822145683456-xvyg?utm_source=share&utm_medium=member_ios

**Reply by clizbe** [reply]
![image](https://github.com/TulipaEnergy/TulipaEnergyModel.jl/assets/11889283/1003ad13-aab0-4960-923b-8ff0cdaa2c04)

**Comment by clizbe** [comment]
Other conferences:
- [JuliaCon](https://juliacon.org/2024/), Eindhoven, July 9th–12th, Deadline: 15 Jan
- [ECOS](https://ecos2024.com/), Greece, 30 June- 5 July, Deadline: 15 Feb
- [AIE Workshop](https://sites.uwasa.fi/aieworkshop/), Finland, 20-22 May, Deadline: 1 Mar
- Maybe for 2025 when we have some studies to show off: [Future of Utilities - Energy Transition Summit](https://futureofutilities.com/events/energy-transition-summit/)

**Comment by clizbe** [comment]
Closing this for now. We can reopen if we want to use it for future conferences/dissemination discussion.
